static/redfish/v1/schema/OemServiceRoot_v1.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+
+  <edmx:Reference Uri="http://docs.oasis-open.org/odata/odata/v4.0/errata03/csd01/complete/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Namespace="Org.OData.Core.V1" Alias="OData"/>
+  </edmx:Reference>
+  <edmx:Reference Uri="http://docs.oasis-open.org/odata/odata/v4.0/errata03/csd01/complete/vocabularies/Org.OData.Measures.V1.xml">
+    <edmx:Include Namespace="Org.OData.Measures.V1" Alias="Measures"/>
+  </edmx:Reference>
+  <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/RedfishExtensions_v1.xml">
+    <edmx:Include Namespace="RedfishExtensions.v1_0_0" Alias="Redfish"/>
+  </edmx:Reference>
+  <edmx:Reference Uri="http://redfish.dmtf.org/schemas/v1/Resource_v1.xml">
+    <edmx:Include Namespace="Resource"/>
+    <edmx:Include Namespace="Resource.v1_0_0"/>
+  </edmx:Reference>
+
+  <edmx:DataServices>
+
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="OemServiceRoot">
+      <Annotation Term="Redfish.OwningEntity" String="IBM"/>
+    </Schema>
+
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="OemServiceRoot.v1_0_0">
+      <Annotation Term="Redfish.OwningEntity" String="IBM"/>
+      <Annotation Term="Redfish.Release" String="1.0"/>  <!--OemServiceRoot Schema is NOT a Redfish Release.-->
+
+      <ComplexType Name="ManagerInformation" BaseType="Resource.OemObject">
```

Comment:
I am surprised this ComplexType is needed or isn't just IBM ?

Reply:
I don't understand the schema well enough to comment.  The properties it describes seem okay to me.

Reply:
So far I have seen all properties in xmls are encapsulated in either a 1) EntityType or 2) ComplexType tag.
I avoided using EntityType because it requires ["a uniquely identifiable key" ](https://www.dmtf.org/sites/default/files/Redfish_School-Introduction_to_CSDL.pdf) that I have not seen implemented. ComplexType seemingly avoided this issue. 
Consistency is not a thing here sadly - so decision will be kind of arbitrary.

Reply:
While we are here, do we want to call this a "Redfish.Release" ? Sorry i keep forgetting to ask this question.
Line 26 is the relevant line here.